src/User/Migration/m000000_000011_add_auth_tf_mobile_phone_column_to_user_table.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Handles adding columns to table `{{%user}}`.
+ */
+class m220708_080521_add_auth_tf_mobile_phone_column_to_user_table extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->addColumn('{{%user}}', 'auth_tf_mobile_phone', $this->string(20)->after('auth_tf_type'));
```

Comment:
Same as above. Why two different migrations? If not strictly necessary, use only one